### PR TITLE
GN-4220: Generic RDFA insert

### PR DESCRIPTION
**Pull request by dkozickis**
First iteration on https://binnenland.atlassian.net/browse/GN-4220 (Generic rdfa variables input plugin), allows to insert RDFA as "plain" HTML.

Exploring folder-tree UI as next step. 

https://github.com/lblod/ember-rdfa-editor-lblod-plugins/assets/769698/7a28aaaf-95a8-416a-b0be-fd5199fa1e11

### CODE REVIEW

**Review by lagartoverde — CHANGES_REQUESTED** [review]
Looks good to me, but I'm having some problems with the placement of the suggestions, they appear on the bottom of the modal for me
Screenshot 2023-06-01 at 15 17 09

File: addon/components/generic-rdfa-variable/insert-menu.hbs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<Toolbar::Button
+  @disabled={{this.disabled}}
```
Comment (elpoelma) [inline addon/components/generic-rdfa-variable/insert-menu.hbs:2]: The `disabled` property does not seem to be defined in the backing class.

**Review by elpoelma — CHANGES_REQUESTED** [review]
Seems to work well overall :) Just a small note.

**Review by abeforgit — CHANGES_REQUESTED** [review]
when opening the modal, the codemirror editor is not focused
when closing the modal (in any way), the editor is not refocused

**Pull request merged**